addons/account/wizard/account_tour_upload_bill.py

```diff
@@ -21,18 +28,80 @@ class AccountTourUploadBill(models.TransientModel):
     )
 
     def _selection_values(self):
+        values = [
+            ('sample', 'Try a sample vendor bill'),
+            ('upload', 'Upload your own bill'),
+        ]
         journal_alias = self.env['account.journal'] \
             .search([('type', '=', 'purchase'), ('company_id', '=', self.env.company.id)], limit=1)
+        if journal_alias.alias_domain:
+            values += [('email', _('Or send a bill to %s@%s', journal_alias.alias_name, journal_alias.alias_domain))]
```

Comment:
@mart-e Is this the good way to do handle the translations for a dynamic selection value? Is there a better way?

Reply:
How are the values to be translated in this case even retrieved actually?

Reply:
@william-andre Yes, this one is correct (but not the `sample` and `upload` ones).
For static selections, they are stored in db (`ir.model.fields.selection`) and are translated like any other record, nothing to be done. For dynamic selections however, the list of selections are evaluated on the fly. Often the selection label is actually a translatable record (e.g. `[(rec.reference, rec.name)]` where `name` is a translatable field) but in the case of static string like you, it should be code translation, using the `_()` method.